docs/how-to-guides/eagleye-integration-guide.md

```diff
@@ -0,0 +1,140 @@
+# Eagleye Integration Guide
+
+This page will show you how to set up [eagleye](https://github.com/MapIV/eagleye), which is integrated into autoware.
+
+Discussions during the integration are as follows.
+https://github.com/orgs/autowarefoundation/discussions/3257
+
+## What is Eagleye?
+
+Eagleye provides a cost-effective alternative to LiDAR and point cloud-based localization by using low-cost GNSS and IMU sensors to provide vehicle position and attitude.
+Eagleye can also compensate for wheel speed and IMU angular velocity bias by using GNSS Doppler velocity.
+
+Autoware users will be able to choose between their existing LiDAR and point cloud-based localization stacks or GNSS/IMU-based eagleye localizer, depending on their specific needs and operating environment.
+
+There are two ways to utilize eagleye results with the autoware localization stack.
+
+1. Feed only Twist into the EKF localizer.
+
+![eagleye twist integration](images/eagleye-integration-guide/eagleye-twist.png)
+
+2. Feed both Twist and Pose from eagleye into the EKF localizer.(twist can also be used with regular gyro_odometry
```

Comment:
L21-24
```suggestion
2. Feed both twist and pose from Eagleye into the EKF localizer (twist can also be used with regular `gyro_odometry`).
![Eagleye pose integration](images/eagleye-integration-guide/eagleye-pose.png)

```

Similar to the previous comment about a numbered list. Also, fix some typos.